docs/pages/Cardinal/CoreConcepts/Reads.mdx

```diff
@@ -0,0 +1,66 @@
+# Reads
+
+Reads define the request and reply types to read game state data. When reads are created, Cardinal automatically sets up an endpoint in the game server to process the read.
+
+## Creating a Read
+
+To create a read, first define a Request struct and a Reply struct:
+
+```go
+
+type PlayerRequest struct {
+    ID string
+}
+
+type PlayerReply struct {
+    Items       []Item
+    Position    Coordinates
+}
+```
+
+Once the request and reply structs have been defined, instantiate the Read. Note that the `NewReadType` function takes a name argument of type string. Cardinal uses this name to automatically create the endpoint for this read. For example, passing "foo-bar" would result in the endpoint
+`address:port/read/foo-bar`. (TODO: update this after we finalize how we want to handle endpoints)
```

Comment:
Add "CAR-139" to this comment so we can later search for it.

https://linear.app/arguslabs/issue/CAR-139/refactor-http-endpoints-to-follow-a-clear-readtx-pattern